gobblin-core/src/main/java/org/apache/gobblin/converter/json/JsonStringToJsonIntermediateConverter.java

```diff
@@ -530,7 +531,7 @@ public static String arrayType(JsonElement schema)
       }
     }
 
-    public static JsonElement getItems(JsonObject object) {
+    public static JsonElement getItemsWithinDataType(JsonObject object) {
```

Comment:
For this one I meant that you can also do `getDataType` within the function, since every call to `getItems` calls `getDataType` first

Reply:
👍